hera_mc/cm_hookup.py

```diff
@@ -62,6 +66,47 @@ def get_hookup(self, hpn_list, rev, port_query, at_date,
         at_date:  date for hookup validity
         exact_match:  boolean for either exact_match or partial
         show_levels:  boolean to include correlator levels
+        force_new:  boolean to force a full database read as opposed to checking file
+        force_specific:  boolean to force this to read/write the file to use the supplied values
```

Comment:
I'm having a bit trouble understanding the purpose of the `force_specific` mode. It writes out a cache file based on the input arguments if I understand things correctly. I imagine that users are unlikely to type in very many arguments manually, so this means that the output file will only contain a few pieces of information, right? Can you add some text to the docstring explaining the intended use case?

Reply:
I want to keep the earlier functionality, so force_specific makes it act just like the earlier version.

Reply:
So the problem is that, if I'm reading the code correctly, if a person runs the program once with `--force-specific`, then reruns it later without that flag, the second invocation will pick up the cache file made by the first one. This file will contain some arbitrary subset of the full hookup rather than the default, full `HH`hookup. It seems like this could be very confusing if one forgot about the caching or if the two invocations were separated widely in time. Should the cache file only be written out if `force_specific` is false?